app/javascript/src/Navigation/components/header/Avatar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
Because [PF4 Avatar](https://patternfly-react.surge.sh/patternfly-4/components/avatar) renders only the User icon, is not ready to receive children. We need the popup menu inside.

Reply:
I see, in any case I'd say we wrap the PF4 Avatar instead of using our own <img> etc. And also I'd use a different name because Avatar should really be just an Avatar. The navigation menu is called _session_ so I'd stick to that name for consistency, same way there is `AccountSettings` and `Document` components.

Reply:
Refactoring